net/mlvpn/Makefile

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (C) 2017 Zhong Jianxin <azuwis@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=mlvpn
+PKG_VERSION:=2.3.2
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/zehome/MLVPN.git
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_VERSION:=6f13423b8108f46edb9f230deee20e3741abe64c
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-$(PKG_SOURCE_VERSION).tar.xz
+PKG_MIRROR_HASH:=e77916143162ffa050bc07482264f726b0c03a8fa9e07bb594a3759f512830cd
+
+PKG_LICENSE:=BSD-2-Clause
```

Comment:
Swap order between PKG_LICENSE and PKG_MAINTAINER.

Add PKG_LICENSE_FILES:=LICENCE